### ISSUE TITLE
generate sequencing_summary file from duplex basecalled reads

### ISSUE THREAD

**Issue by sarahszaffich** [nanoporetech/dorado#222]
Hi, I used dorado duplex on pod5 files and tried dorado summary to generate a sequencing_summary file based on the duplex basecalled bam file. Unfortunately, I got this error message: [error] > Cannot generate sequencing summary for files with no RG tags

generating sequencing_summary files based on simplex basecalled data worked without any problems. What am I doing wrong for duplex results?

Thanks!

**Comment by vellamike** [comment]
Hi @sarahszaffich - this is a bug we are aware of which has been fixed internally and will shortly be in an upcoming release.

**Comment by Faewks** [comment]
@vellamike 
do you know approximately when this release will happen?
Best regards.

**Comment by iiSeymour** [comment]
@Faewks we are looking at releasing tomorrow.

**Comment by Faewks** [comment]
@iiSeymour 
That's great! 
Thank you very much.

**Comment by tijyojwad** [comment]
Hi @Faewks - this is now available in v0.3.1 https://github.com/nanoporetech/dorado/releases/tag/v0.3.1

**Comment by nservant** [comment]
Hi there,
Sorry but I still have the same issue with the version 0.3.1.  
I do not have any `RG` tag in the Dorado BAM file. Is there any additional parameter to specify on the Dorado command ?

**Comment by iiSeymour** [comment]
@nservant did you basecall with v0.3.1 or just run `dorado summary` with calls from v0.3.0?

**Comment by nservant** [comment]
I did both

**Comment by nservant** [comment]
I mean basecaling with 0.3.1 and summary with the same version. But indeed, I have no RG tag in the output BAM file of the basecaller

**Comment by nservant** [comment]
here is the BAM header I have 

```
samtools view -H 20230426_1757_X2_FAW71870_210e7908.bam 
@HD	VN:1.6	SO:unknown
@PG	ID:basecaller	PN:dorado	VN:0.3.1+bb8c5ee	CL:dorado basecaller -x cuda:0 dna_r10.4.1_e8.2_400bps_fast@v4.1.0 fast5_pass
@PG	ID:samtools	PN:samtools	PP:basecaller	VN:1.15.1	CL:/usr/local/conda/envs/samtools-1.15.1/bin/samtools view -H 20230426_1757_X2_FAW71870_210e7908.bam
```

and the dorado command line ;

```
dorado basecaller -x cuda:0 \
                  dna_r10.4.1_e8.2_400bps_fast@v4.1.0 \
                  fast5_pass/ > 20230426_1757_X2_FAW71870_210e7908.bam
```

**Comment by tijyojwad** [comment]
Hi @nservant - are you running `dorado` on fast5 data? Unfortunately we don't support read group generation w/ fast5, only pod5s. Can you convert your dataset to pod5 and give it another go?

**Comment by nservant** [comment]
Yes, this is what I had in mind ... I will convert them in pod5.
I think you should clarify this somewhere in the doc, because it means that with fast5 data, you have no RG tag, and so no summary ...

**Comment by nservant** [comment]
confirmed. I works well from pod5. Thanks

**Comment by tijyojwad** [comment]
Yes agreed, will add to our documentation

**Comment by Faewks** [comment]
> Hi @Faewks - this is now available in v0.3.1 https://github.com/nanoporetech/dorado/releases/tag/v0.3.1

Thank you, it does work from pod5 data.
